projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl

```diff
@@ -211,7 +211,7 @@ VDK distribution docker repository secret name
 {{- end }}
 
 {{- define "shouldCreatePipelinesControlServiceDockerRepoSecret" }}
-  {{- if and (eq .Values.image.registryType "generic") .Values.image.registryUsernameReadOnly .Values.image.registryPasswordReadOnly }}
+  {{- if and (.Values.image.registryUsernameReadOnly) (.Values.image.registryPasswordReadOnly) }}
```

Comment:
@gageorgiev  there was no property called .Values.image.registryType. That is why it was failing.